tasks/debian.yml

```diff
@@ -10,6 +10,23 @@
       - python3-pip
       - python3-setuptools
     state: present
+  when:
```

Comment:
It looks like
```
apt-get install python3 python3-distutils python3-venv
```
works on 18.04 (3.6) and 20.04 (3.8) and would avoid having a conditional. It would also mean people can use non-LTS versions (at their own risk obviously!)

Reply:
:+1: using the default `python3*` packages for the given operating system makes sense. These are also the versions used for building the Ice Python wheels used e.g. in https://github.com/ome/ansible-role-omero-server/pull/55/files

@SteveO-sudo do you want to try and push a commit to install the packages suggested by @manics?

Reply:
I made the changes suggested by @manics. NB this role fails in both 18.04 and 20.04 on fresh installations with a "No package matching python3-venv is available" UNLESS an "apt update" to refresh the package information is performed first.

Reply:
Thanks for the testing @SteveO-sudo. In this case, is it worth setting [update_cache](https://docs.ansible.com/ansible/latest/collections/ansible/builtin/apt_module.html#parameter-update_cache) to `yes` to handle this scenario?

Reply:
Thanks for the suggestion. Done.